### ISSUE TITLE
Plugins added to chain are using clients CPU a lot

### ISSUE THREAD

**Issue by cehatoptkn** [apohl79/audiogridder#8]
I am trying to add a few heavy plugins in one chain - every added plugin increases clients CPU usage, as well as a server usage. Network usage was not increased.

i was trying it with Waves Krammer Tape - in 96k it and uses ~+2% of CPU every added plugin, i have added like 7-8 of them in one chain.

UPD: My issue - i've tried to do it in 0 Buffer. As i understand it working like this when the plugin self latency is more then buffer. t will be a very good feature to see the self latency of every added plugin and count it in the top, to see when you need to increase the buffer.

**Comment by apohl79** [comment]
I can't reproduce this in logic or cubase. The load increases when starting the playback, as there is some work the client has to do, but is does not increase when adding remote plugins. The load does also not compare to the load caused by the DSP processing of the plugins added. 

As an example (running on a 2019 macbook pro 13"), I added 10 AG instances on a track in cubase. When starting the playback, the CPU load for cubase increased by around 10% from ~15% without AG to ~25% with AG. The inserts included between 1 to 3 VST3 and AU plugins from different vendors. Adding or removing remote plugins did not affect the load.

In reaper though I saw an increase of maybe ~2% with each added AG instance even without starting the playback. So it looks like this is related to reaper. It also did not matter in reaper if I added the AU or VST3 version of AG.

What DAW did you test with?
